lib/beaker/host/cisco.rb

```diff
@@ -70,8 +70,12 @@ def package_config_dir
     # @return [String] Command string as needed for this host
     def prepend_commands(command = '', user_pc = '', opts = {})
       return user_pc unless command.index('vsh').nil?
+      if self[:platform] =~ /cisco_nexus/
```

Comment:
@tvpartytonight yeah, I believe this is true, that there are platform strings like `cisco_exr` and such that would match the host object & not match `cisco_nexus`. But those have changed over time, so I'm not sure if that's gone away. @LuvCurves ?